lit-base/model_loader.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+import mxnet as mx
+
+import tempfile
+from urllib import urlretrieve
+
+
+# load model
+def load_model(s_fname, p_fname):
```

Comment:
follow the documentation standard similar to 
https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/executor.py#L113